en/components/grids_templates/keyboard_navigation.md

```diff
@@ -22,35 +22,77 @@ _keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI
 
 ## @@igComponent Keyboard Navigation
 
-Keyboard navigation is available by default in any grid and aims at covering as many as possible features and scenarios for the end user. When you focus a specific cell and press one of the following key combinations, the described behavior is performed.
+Keyboard navigation is available by default in the @@igxName and aims at covering as many as possible features and scenarios for the end user. Now in order to improve the overall page navigation experience we reduced the tab stops in the grid. Currently we have only the following main tab stops:
+    - GroupBy / Toolbar area /if is enabled/;
+    - header of the grid;
+    - body of the grid;
+    - column summaries /if are enabled/;
+    - the paginator /if is enabled/;
 
-### Key combinations
-@@if (igxName === 'IgxHierarchicalGrid') {
- - `Arrow Up` - navigates one cell up, going up the grid hierarchy if necessary (no wrapping);
- - `Arrow Down` - navigates one cell down, going deeper into the grid hierarchy if necessary (no wrapping);}@@if (igxName === 'IgxGrid' || igxName === 'IgxTreeGrid') {
- - `Arrow Up` - navigates one cell up (no wrapping);
- - `Arrow Down` - navigates one cell down (no wrapping);}
+> Note:
+> Due to this behavior change to navigate between the cells with `tab` and `shift + tab` key is no longer supported in the @@igxName.
```

Comment:
Due to the newly exposed behavior the navigate between the cells with `tab` and `shift` + `tab` key are no longer supported in the @@igxName. Having that in mind, when pressing `tab` key you are going to experience the following sequence navigation: `Grid Toolbar` -> `Grid Headers` -> `Grid Body` -> `Summaries` -> `Footer / Grid Paginator`. Also, keep in mind that if there is a focusable element in the @@igxName template it will be included in the sequence of the page tabs.